portfolio/src/app/case-studies/page.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import Image from 'next/image'
+
+import { CarRentDesktop } from '../../../assets'
+
+const page = () => {
+  return (
+    <div className=''>
```

Comment:
This 'div' with an empty className can be removed. It would be better to provide a meaningful class name if it is needed for styling purposes.